frontend/WikiContrib-Frontend/src/query.js

```diff
@@ -796,18 +797,26 @@ export class Query extends Component {
                           )}
                           <br />
                           <Card.Content extra>
-                            <div>
-                              <label>
+                            <Popup
+                              content=<a href="https://github.com/wikimedia/WikiContrib"
+                                rel="noopener noreferrer" target="_blank">
+                                Contact The maintainers of this Project to learn more about this feature
```

Comment:
Change the message to "This feature is currently disabled. Contact the tool maintainers to learn more."